src/main/java/com/zenjava/javafx/maven/plugin/JarMojo.java

```diff
@@ -144,7 +147,31 @@
      *
      * @parameter property="jfx.copyAdditionalAppResourcesToJar" default-value="false"
      */
-    private boolean copyAdditionalAppResourcesToJar = false;
+    protected boolean copyAdditionalAppResourcesToJar = false;
+
+    /**
+     * To skip copying all dependencies, set this to true. Please note that all dependencies will be added to the
+     * manifest-classpath as normal, only the copy-process gets skipped.
+     *
+     * @since 8.8.0
+     *
+     * @parameter property="jfx.skipCopyingDependencies"
+     */
+    protected boolean skipCopyingDependencies = false;
+
+    /**
+     * @since 8.8.0
+     *
+     * @parameter property="jfx.useLibFolderContentForManifestClasspath" default-value="false"
+     */
+    protected boolean useLibFolderContentForManifestClasspath = false;
+
+    /**
+     * @since 8.8.0
+     *
+     * @parameter property="jfx.fixedManifestClasspath" default-value=""
+     */
+    protected String fixedManifestClasspath = null;
```

Comment:
At least for the integration with the Drombler FX Maven Plugin useLibFolderContentForManifestClasspath should be enought. Do you have a use case for this property? The trouble with such a property is that it has to be formatted correctly (using "/" and " " as delimiters for file names and path entries). You can't just use the output from the Maven Dependency Plugin for instance, as it is platform specific and not a Manifest classpath.